metrics/bleu/bleu.py

```diff
@@ -84,13 +87,27 @@
 """
 
 
+@dataclass
+class BleuConfig(evaluate.info.Config):
+
+    name: str = "default"
+
+    tokenizer: Optional[Callable] = None
+    max_order: int = 4
+    smooth: bool = False
+
+
 @evaluate.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
 class Bleu(evaluate.Metric):
-    def _info(self):
+    CONFIG_CLASS = BleuConfig
+    ALLOWED_CONFIG_NAMES = ["default"]
```

Comment:
I guess this line can be made optional when it's only `["default"]` ?

Reply:
You mean to put it there explicitly? Indeed, I just thought it would make it a bit more clear for users that there is a default. Happy to delete it if you think it's more confusing.

Reply:
Indeed it's maybe better to be explicit here actually